build.override.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="WcfSetup" Condition="'$(WithCategories)' == 'OuterLoop'" BeforeTargets="Test">
+    <Exec Command="$(MySetupScript) > $(MyLogFile)" />
+    <ReadLinesFromFile File="$(MyLogFile)" >
```

Comment:
Why do we need to read from the log?  Was this something CoreFx requested?  The Exec task exposes ExitCode directly if that's all we need.  See https://msdn.microsoft.com/en-us/library/x8zx72cd.aspx